packages/tools/webpack-fluid-loader/README.md

```diff
@@ -58,6 +62,7 @@ or in an optional `config.json` file in the `baseDir` passed into `webpack-fluid
 {
     "fluid": {
         "webpack": {
+            "discoveryEndpoint": "https://discoveryEndpoint.com",
```

Comment:
nit: urls are case-insensitive, so make it a lower-case `e`

Reply:
Got it. Thanks. Done.